### PR TITLE
refactor: change capacity to shannon

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: lib/ckb/utils.rb [hunk 1]
```diff
@@ -23,5 +23,12 @@ def self.delete_prefix(str)
 
       str
     end
+
+    # @param capacity [Integer] Byte
+    #
+    # @return [Integer] shannon
+    def self.to_shannon(capacity)
```
Comment (ashchan) [inline lib/ckb/utils.rb]: How about `byte_to_shannon`?

File: test/ckb/tx/output_test.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ def test_calculate_min_capacity
       }
     )
 
-    min_capacity = 41
+    min_capacity = 41 * 10**8
```
Comment (ashchan) [inline test/ckb/tx/output_test.rb]:
```suggestion
    min_capacity = Ckb.Utils.to_shannon(41)
```

File: lib/ckb/utils.rb (hunk 1 above)
Comment (classicalliu) [reply lib/ckb/utils.rb]: updated

File: lib/ckb/utils.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 require 'secp256k1'
 
 module Ckb
-  MIN_CELL_CAPACITY = 40
-  MIN_UDT_CELL_CAPACITY = 48
+  MIN_CELL_CAPACITY = 40 * 10**8
```
Comment (xxuejie) [inline lib/ckb/utils.rb]: Can we use the utility function here as well?

Comment (classicalliu) [reply lib/ckb/utils.rb]: OK

Comment (classicalliu) [reply lib/ckb/utils.rb]: updated, and move to bottom after `Utils` defined

**Review by ashchan — APPROVED** [review]

**Review by xxuejie — APPROVED** [review]

**Pull request merged**